src/pages/membership/components/createYouthProfileForm/CreateYouthProfileForm.tsx

```diff
@@ -116,42 +118,15 @@ function CreateYouthProfileForm(props: Props) {
     <Formik
       initialValues={{
         ...props.profile,
-        address: '',
-        postalCode: '',
-        city: '',
-        phone: '',
-        schoolName: '',
-        schoolClass: '',
-        language: '',
-        approverFirstName: '',
-        approverLastName: '',
-        approverEmail: '',
-        approverPhone: '',
+        birthDate: format(new Date(props.profile.birthDate), 'dd.MM.yyyy'),
         terms: false,
       }}
       initialErrors={{
         terms: 'validation.required',
       }}
-      onSubmit={values => {
-        props.onValues({
-          firstName: values.firstName,
-          lastName: values.lastName,
-          address: values.address,
-          postalCode: values.postalCode,
-          city: values.city,
-          email: props.profile.email,
-          phone: values.phone,
-          birthDate: format(new Date(props.profile.birthDate), 'yyyy-MM-dd'),
-          schoolName: values.schoolName,
-          schoolClass: values.schoolClass,
-          approverFirstName: values.approverFirstName,
-          approverLastName: values.approverLastName,
-          approverPhone: values.approverPhone,
-          approverEmail: values.approverEmail,
-          languageAtHome: values.languageAtHome,
-          photoUsageApproved: values.photoUsageApproved,
-        });
-      }}
+      onSubmit={(values: FormValues) =>
+        props.onValues({ ...values, birthDate: props.profile.birthDate })
```

Comment:
This is fine by me, but I think that you should not have the `birthDate` value within the form state at all. You are currently using the form's model as a kind of view layer because you are using it to display a human readable date. When finally sending the values, you have to overwrite the data.

You could remove it entirely from `Formik` and just have the input reflect the value it receives from the API directly.

Reply:
Removed `birthDate` from form data and passed value directly to TextField.